.github/workflows/cd.yml

```diff
@@ -30,4 +38,13 @@ jobs:
           credentials: ${{ secrets.FLOW_HOSTING_PROD_SA }}
           path: ./
           glob: 'version.txt'
-          destination: flow-cli/
\ No newline at end of file
+          destination: flow-cli/
+  homebrew:
+      name: Bump Homebrew formula
+      runs-on: ubuntu-latest
+      steps:
+        - uses: mislav/bump-homebrew-formula-action@v1
+          with:
+            formula-name: flow-cli
+          env:
+            COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
```

Comment:
I don't see `COMMITTER_TOKEN` in the secrets, are we missing it?

Reply:
Yes, I have to set it up, but I didn't yet found the time, I will do that before it is merged, should've left it as draft PR.

Reply:
@Kay-Zee  Actually looking at it, it seems that I don't have enough permissions to set repo secrets. Can you give me help with this?